README.md

```diff
@@ -1,57 +1,23 @@
-[![Build Status](https://travis-ci.org/meteorhacks/kadira.svg?branch=master)](https://travis-ci.org/meteorhacks/kadira)
+# Meteor APM (formerly Kadira)
+## [Performance Monitoring for Meteor Apps on Galaxy](https://www.meteor.com/hosting) 
 
-## [Kadira - Performance Monitoring for Meteor](https://kadira.io) 
-
-[![Kadira - Performance Monitoring for Meteor](https://i.cloudup.com/LwrCCa_RRE.png)](https://kadira.io)
+[![APM - Performance Monitoring for Meteor Apps on Galaxy](https://i.cloudup.com/LwrCCa_RRE.png)](https://www.meteor.com/hosting)
 
 ### Getting started
 
-1. Create an account at <https://kadira.io>
-2. From the UI, create an app. You'll get an `AppId` and an `AppSecret`.
-3. Run `meteor add meteorhacks:kadira` in your project
-4. Configure your Meteor app with the `AppId` and `AppSecret` by adding the following code snippet to a `server/kadira.js` file:
-
-```js
-Meteor.startup(function() {
-  Kadira.connect('<AppId>', '<AppSecret>');
-});
-```
-
-Now you can deploy your application and it will send information to Kadira. Wait up to one minute and you'll see data appearing in the Kadira Dashboard.
-
-
-### Auto Connect
-
-Your app can connect to Kadira using environment variables or using [`Meteor.settings`](http://docs.meteor.com/#meteor_settings).
-
-#### Using Meteor.settings
-Use the followng `settings.json` file with your app:
-
-```js
-{
-  ...
-  "kadira": {
-    "appId": "<appId>",
-    "appSecret": "<appSecret>"
-  }
-  ...
-}
-```
-
-The run your app with `meteor --settings=settings.json`.
-
-#### Using Environment Variables
+1. If you don't already have a Galaxy account, sign up [here](https://www.meteor.com/hosting).
+2. Run `meteor add mdg:meteor-apm-agent` inside your Meteor project.
+3. Once you have deployed your app, enable "Galaxy Professional" from the app settings page.
 
-Export the following environment variables before running or deploying your app:
+Now your app will send information to Meteor APM. Visit [Galaxy](https://galaxy.meteor.com) to find links to your APM dashboard.  Links can be found in the following places:
+- In the *Performance* area on the App Overview & Containers pages.
+- In the *Galaxy Professional* area on the App Settings page.
 
-```
-export KADIRA_APP_ID=<appId>
-export KADIRA_APP_SECRET=<appSecret>
-````
+Wait up to one minute and then data should begin appearing.
 
 ### Error Tracking
```

Comment:
Please remove this section on Error Tracking. It's one of the features and we don't need to call it out.